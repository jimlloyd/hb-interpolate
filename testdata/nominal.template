// nominal.template
{{name}}
{{repository.url}}
{{#if author.name}}
{{author.name}} <mailto:{{author.name}}>
{{else}}
{{author}}
{{/if}}
