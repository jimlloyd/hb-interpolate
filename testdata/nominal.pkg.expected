// nominal.template
hb-interpolate
git+https://github.com/jimlloyd/hb-interpolate.git
Jim Lloyd <mailto:Jim Lloyd>
